src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -15,47 +16,91 @@
     private static final String FlightLogFormat = "Operation: {}, flightClass: {}, flightId: {}, timestamp: {}";
     private static final String StepLogFormat = "Operation: {}, flightClass: {}, flightId: {}, stepClass: {}, " +
         "stepIndex: {}, direction: {}, timestamp: {}";
+    // Id of the flight
+    private static final String FLIGHT_ID_KEY = "flightId";
+    // Class of the flight
+    private static final String FLIGHT_CLASS_KEY = "flightClass";
+    // Class of the flight step
+    private static final String FLIGHT_STEP_CLASS_KEY = "flightStepClass";
+    // Direction of the step (START, DO, UNDO or SWITCH)
+    private static final String FLIGHT_STEP_DIRECTION_KEY = "flightStepDirection";
+    // The step's execution order
+    private static final String FLIGHT_STEP_NUMBER_KEY = "flightStepNumber";
+    // The type of operation primarily being logged (startFlight, startStep, endStep or endFlight)
+    private static final String FLIGHT_OPERATION_KEY = "flightStepOperation";
+    private static final String FLIGHT_OPERATION_START = "startFlight";
+    private static final String FLIGHT_OPERATION_END = "endFlight";
+    private static final String FLIGHT_STEP_OPERATION_START = "startStep";
+    private static final String FLIGHT_STEP_OPERATION_END = "endStep";
```

Comment:
Thoughts on sticking these in a string enum? Would avoid some of the lengthy duplication of `private static final String <...>`

Reply:
I was wondering about that but the static string doesn't bug me too much they're not really logically enums

Reply:
To me the `_KEY` fields conceptually seem like an enum as they're all key values for the map. The other fields are text values which I agree are not locally enums.